openapi/openapi.yaml

```diff
@@ -58,4 +60,23 @@ paths:
     $ref: 'paths/lambdas/lambdas@collections@wearables.yaml'
   /lambdas/collections/wearables-by-owner/{address}:
     $ref: 'paths/lambdas/lambdas@collections@wearables-by-owner@{address}.yaml'
+  /lambdas/contentv2/scenes:
+    $ref: 'paths/lambdas/lambdas@contentv2@scenes.yaml'   
+  /lambdas/contentv2/parcel_info: 
```

Comment:
Personal opinion: This shouldn't be another endpoint, the `cids` should be another query parameter in `/lambdas/contentv2/scenes`. Anyway, the documentation for it is okay, changing that would mean a refactor of lambdas code.